sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/DescribeTableExec.scala

```diff
@@ -38,12 +38,28 @@ case class DescribeTableExec(
     addSchema(rows)
 
     if (isExtended) {
```

Comment:
I checked how we describe v1 table
```
-- !query 10
DESC EXTENDED t
-- !query 10 schema
struct<col_name:string,data_type:string,comment:string>
-- !query 10 output
a                   	string              	                    
b                   	int                 	                    
c                   	string              	                    
d                   	string              	                    
# Partition Information	                    	                    
# col_name          	data_type           	comment             
c                   	string              	                    
d                   	string              	                    
                    	                    	                    
# Detailed Table Information	                    	                    
Database            	default             	                    
Table               	t                   	                    
Created Time [not included in comparison]
Last Access [not included in comparison]
Created By [not included in comparison]
Type                	MANAGED             	                    
Provider            	parquet             	                    
Num Buckets         	2                   	                    
Bucket Columns      	[`a`]               	                    
Sort Columns        	[`b`]               	                    
Comment             	table_comment       	                    
Table Properties    	[t=test, e=3]       	                    
Location [not included in comparison]/{warehouse_dir}/t	                    
Storage Properties  	[a=1, b=2]          	                    
Partition Provider  	Catalog
```

maybe we can follow it
```
if (extended) {
  addPartitioning(rows)
  rows += toCatalystRow("# Detailed Table Information", "", "")
  // display namespace, comment, location, properties, etc.
}
```

Reply:
@cloud-fan 
updated, please help review again. 
thanks.